groups/groups.yaml

```diff
@@ -57,14 +57,18 @@ groups:
   - email-id: calendar-maintainers@knative.team
     name: calendar-maintainers
     description: |-
-      Folks that can manage Knative calendar
+      Folks that can manage Knative calendar for one-off calendar invites
     settings:
       AllowExternalMembers: "false"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
     owners:
-      - omerbensaadon@knative.team
       - pmorie@knative.team
       - thisisnotapril@knative.team
+      - anassi@google.com
+      - csantanapr@knative.team
+      - csantana23@gmail.com
+      - lball@knative.team
       - vaikas@knative.team
-    members:
```

Comment:
Just curious why this removal?

Reply:
Made everyone owner instead of members, I tried to mimic the other groups.yaml files

I'm either fine owners or members

Reply:
Maybe I'm reading the diff wrong, but it seems like abrennan has been now removed?